arxivdigest/api/validator.py

```diff
@@ -90,6 +91,28 @@ def topic_recommendation(json):
             return err
     return None
 
+def duplicate_topic_suggestion(json):
```

Comment:
should use same logic for blocking as get_user_feedback_topics such that we ensure consistency